frontend/src/components/pages/rp-connect/onboarding/add-user-step.tsx

```diff
@@ -144,6 +159,27 @@ export const AddUserStep = forwardRef<UserStepRef, AddUserStepProps & MotionProp
     );
 
     const { createUserWithSecrets, isPending } = useCreateUserWithSecretsMutation();
+    const { mutateAsync: createSecret } = useCreateSecretMutation({ skipInvalidation: true });
```

Comment:
I think it's worthwhile not to skip invalidation, since you may want to give feedback to the user that this new secret is already present?